api/report.py

```diff
@@ -45,6 +46,7 @@ def post(self):
         campaign.save()
         return new_report.dict()
 
+@DeprecationWarning
```

Comment:
Why is this here?

Reply:
I just mark that this class will be removed since we don't need my report page anymore.